schema/var_type.go

```diff
@@ -1,29 +1,25 @@
 package schema
 
 import (
-	"bytes"
 	"fmt"
 	"strings"
 )
 
 type VarType struct {
-	expr string
-	Type DataType
+	Expr string   // Type, ie. map<string,map<string,uint32>> or []User
+	Type DataType // Kind, ie. map or struct
```

Comment:
@pkieltyka Perhaps we could change the field names as follows (for v2):

```suggestion
	Type string   // ie. map<string,map<string,uint32>> or []User
	Kind Kind // ie. map or struct
```